src/IceRpc.Protobuf/InvokerExtensions.cs

```diff
@@ -73,33 +60,216 @@ public static class InvokerExtensions
             throw;
         }
 
-        // ReadResponseAsync is responsible for disposing the request
-        return ReadResponseAsync(responseTask, request);
+        // ReceiveResponseAsync is responsible for disposing the request
+        return ReceiveResponseAsync(messageParser, responseTask, request, cancellationToken);
+    }
+
+    /// <summary>Sends a request to a service and decodes the response.</summary>
```

Comment:
It would be nice for the summaries to explain the different behavior between `ClientStreamingInvokeAsync`, `ServerStreamingInvokeAsync`, etc.